static/src/javascripts/projects/common/modules/experiments/tests/contribs-banner-us-eoy-impeachment-regulars.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import { getSync as geolocationGetSync } from 'lib/geolocation';
+import { acquisitionsBannerUsEoyTemplate } from 'common/modules/commercial/templates/acquisitions-banner-us-eoy';
+import { getArticleViewCountForWeeks } from 'common/modules/onward/history';
+
+// User must have read at least 6 articles this year to qualify
+const minArticleViews = 6;
```

Comment:
any reason for raising to 6?

Reply:
the copy indicates "you've read more than..." so i'm subtracting 1 and ensuring the smallest number that shows up in copy is still "5"

Reply:
ah, very precise!

Reply:
annoyingly accurate™️